### DISCUSSION TITLE
Guidance for isolated .net5 Azure Functions

### DISCUSSION THREAD

**Discussion by isaacbromberg** [arcus-azure/arcus.observability#240] [General]
In .net 5 functions in isolated mode, there is HostBuilder instead IFunctionsHostBuilder and it should work with standard asp.net core packages but guidance would be helpful instead of personal experiments with unsure results :)

**Comment by tomkerkhove** [comment]
I don't have experience with the isolated worker but I do have with non-Function workloads.

What issues are you having or what guidance are you seeking?

**Comment by isaacbromberg** [comment]
the same as for asp.net core, 'classic' Functions, etc. Example packages set. Example code for Startup and Program examples.

**Reply by tomkerkhove** [reply]
But isn't that already the same for what we have today for non-Functions then? I'm not sure if I get the problem/gap 🤔
